README.rst

```diff
@@ -17,15 +17,21 @@
 .. |Coverage| image:: https://codecov.io/gh/scverse/anndata/branch/master/graph/badge.svg?token=IN1mJN1Wi8
    :target: https://codecov.io/gh/scverse/anndata
 
+.. image:: docs/_static/img/anndata_schema.svg
+   :align: right
+   :width: 350px
+
+.. after image
 
 anndata - Annotated data
 ========================
 
 anndata is a Python package for handling annotated data matrices in memory and on disk, positioned between pandas and xarray. anndata offers a broad range of computationally efficient features including, among others, sparse data support, lazy operations, and a PyTorch interface.
 
+* Discuss development on `GitHub <https://github.com/scverse/anndata>`_.
```

Comment:
The only “issue” with this PR is that this list item is quite redundant in the README.

However, only adding it to the docs index and `include`ing the rest of the bullet points would create two bullet lists.